.github/pull_request_template.md

```diff
@@ -23,7 +23,16 @@ Build:
 -   [ ] `npm run dist`
 -   [ ] Commit resulting `dist/` changes
 
-Release:
+Release (Optimove Web SDK):
```

Comment:
could have multiple templates, but to select one need to manually (?) add query parameter when creating template?